common/utils.go

```diff
@@ -50,6 +50,51 @@ func C2GoArray(str string) []byte {
 	return ret
 }
 
+// GoArray2C transforms a byte slice into its hexadecimal string representation.
```

Comment:
We need to move away of this package cc @joestringer

Reply:
Filed https://github.com/cilium/cilium/issues/11293